README.md

```diff
@@ -66,7 +65,7 @@ target_os = []
 Common build options shared by Windows and Linux:
   - By default `x86|Debug` library will be created. Specify `--arch x64` if you want to build x64 libraries; Specify `--scheme release` if release version of library is to be built.
   - The built binary will be under path specified by `--output_path`. If `--output_path` is not set, the built binary will be under `src/out` directory.
-  - The optional `--ssl_root` should be set to the root directory of lastest OpenSSL 1.1.1 binary. If specified, SDK will link to external openssl library instead of boringssl.
+  - The optional `--ssl_root` should be set to the root directory of lastest OpenSSL 3.0 series directory. If specified, build tool will search external OpenSSL headers in `ssl_root/include` and OpenSSL binaries in `ssl_root/bin`. But binaries are not included in OWT SDK, so applications still need to link libcrypto and libssl.
```

Comment:
Are you suggesting we only support dynamic version of ssl library? (as you renamed the mergelib method to copylib method.

Reply:
No, it can be a static lib. Renaming because we don't need lib.exe for combining owt.lib and ssl libs.

I'm still working on this change because I just found it uses my system's OpenSSL headers instead of the one specified in ssl_root. I'll update this change soon.

Reply:
Updated. But if `ssl_root/include` doesn't have OpenSSL headers, it searches system include directory then. It looks like it's unavoidable unless setting `use_sysroot` to `true`.